pkg/operatorsource/handler_test.go

```diff
@@ -32,13 +39,16 @@ func TestHandle_PhaseHasChanged_UpdateExpected(t *testing.T) {
 		return cacheReconciler
 	}
 
-	handler := operatorsource.NewHandlerWithParams(kubeclient, writer, factory, transitioner, newCacheReconcilerFunc)
+	handler := operatorsource.NewHandlerWithParams(fakeclient, writer, factory, transitioner, newCacheReconcilerFunc)
 
 	ctx := context.TODO()
 
 	// Making two OperatorSource objects that are not equal to simulate a change.
 	opsrcIn, opsrcOut := helperNewOperatorSourceWithEndpoint("marketplace", "foo", "remote"), helperNewOperatorSourceWithEndpoint("marketplace", "foo", "local")
 
+	// Add OperatorSource to fakeclient
+	assert.NoError(t, fakeclient.Create(ctx, opsrcIn));
```

Comment:
Why do an assert here? This isn't actually testing our code, right?

Reply:
That's true. `fakeclient.Create` returns an error object, so I figured I might as well. Since it shouldn't throw an error and it's part of the test setup, I can call `Create` without the assert.